tests/unit/amo/sagas/testReviews.js

```diff
@@ -354,11 +434,10 @@ describe(__filename, () => {
 
       _fetchUserReviews();
 
-      const errorAction = errorHandler.createErrorAction(error);
-      await sagaTester.waitFor(errorAction.type);
+      const expectedAction = errorHandler.createErrorAction(error);
+      const action = await sagaTester.waitFor(expectedAction.type);
```

Comment:
that's cool!

Reply:
It's [undocumented](https://github.com/wix/redux-saga-tester/pull/49). Sigh. We should probably do a sweep and convert all the old saga test suites. A lot of them are already using it.

Reply:
Yep, and probably fork and maintain our own `SagaTester` lib...